puzzles/models.py

```diff
@@ -58,6 +58,11 @@ class Puzzle(models.Model):
 
     active_users = models.ManyToManyField('accounts.Puzzler', related_name="active_puzzles")
 
+    class Meta:
+        constraints = [
+                models.UniqueConstraint(fields=['name', 'hunt'], name='unique_names'),
```

Comment:
nit: Consider "unique_names_per_hunt" - if it seems too verbose, don't worry about it.